src/components/ToolbarButton/ToolbarButton.stories.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import * as React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { ToolbarButton } from "./ToolbarButton";
+
+export default {
+  title: "Atoms/ToolbarButton",
+  component: ToolbarButton,
+  args: {
+    onClick: () => console.log("hei"),
```

Comment:
`console.log` should be used for debugging only, and ESLint will not allow it. Use `console.info`, `console.warn`, or `console.error` for intentional logging.

```suggestion
    onClick: () => console.info("Toolbar button clicked"),
```

Reply:
Yes, I noticed it and will change it